x-pack/plugins/ml/public/alerting/anomaly_detection_alerts_table/render_cell_value.tsx

```diff
@@ -90,9 +90,19 @@ export function getAlertFormatters(fieldFormats: FieldFormatsRegistry) {
           </>
         );
       case ALERT_ANOMALY_SCORE:
+        let latestValue: number | undefined;
+        if (typeof value === 'number') {
+          latestValue = value;
+        } else {
+          const resultValue: number[] = value.split(',').map(Number);
+          latestValue = resultValue.at(-1) as number;
+        }
+        if (latestValue === undefined) {
```

Comment:
It looks like `latestValue` will never be `undefined` here because of the conversion to `Number`. 
Can `value` be an empty string here? If so, should we be converting to a number? as it'll get the value `0` which may be misleading.

Reply:
It's about type signature of the `.at` method. Updated here https://github.com/elastic/kibana/pull/174919/commits/b15f50f0d734e243338bc970a7b69c7af55523e4

Reply:
Maybe I'm not getting something, but I still don't see how it'll ever be hit.
If `value` is a number, `latestValue` will be a number.
If `value` is a string, the split and `map(Number)` will ensure all values are a number. even if `value` is an empty string, the split and map will return the number `0`

Reply:
yes, the `.at` type signature was a bit misleading. Also it's never an empty string. I've updated the code and added unit tests [here](https://github.com/elastic/kibana/pull/174919/commits/463b62278bc0e604f978b9e2f2edfc4fb0f0e70f)